ngsild-payloads/userProfile.jsonld

```diff
@@ -3,7 +3,10 @@
     "type": "UserProfile",
     "userInfos": {
         "type": "JsonProperty",
-        "value": {}
+        "value": {
+            "firstname": "Obi",
+            "lastname": "Wan"
+        }
```

Comment:
I think `firstName` and `lastName` would be better

Reply:
good

Reply:
we will need the email too to send the welcome email